src/main.rs

```diff
@@ -1,3 +1,204 @@
-fn main() {
-    println!("Hello, world!");
+use dashmap::DashMap;
```

Comment:
I saw this being used in an example `tower-lsp` LSP server implementation, I guess it's a "concurrent hash-map" (so even "mutable" methods take `&self`)?